### ISSUE TITLE
Acl, users and groups in a plugin

### ISSUE THREAD

**Issue by darxmac** [cakephp/acl#72]
I have my Users and Groups models and other logic in a plugin, when trying to implement Acl saving new users works, but trying to save changes it fails with a 

```php
AclNode::node() - Couldn't find Aros node identified by "Array ( [Aros0.model] => PUsers.Users [Aros0.foreign_key] => 11 ) "
```

error

The model in the Aros table is just saved as `Users`

What would be the fix, save the full Plugin.Model in the `model` field, or fix that node lookups just use the model name without the plugin part ?

**Comment by darxmac** [comment]
the fix in pr #73 work for me for now

**Comment by rchavik** [comment]
Closing since 73 has been merged
